### DISCUSSION TITLE
How to select text by shift+left/right like traditional ?

### DISCUSSION THREAD

**Discussion by ForteDexe** [kovidgoyal/kitty#4393] [Q&A] [answered]
Right now shift+left=C, shift+right=D. I tried to google but have no result.

**Comment by page-down** [comment, marked as answer]
- Currently kitty cannot select scrollback text via shortcut keys.
- If you need to select text by `shift+left` when typing commands in the shell, you should configure the shell with the shortcuts.
- If you are using a text editor or other program, you should configure the shortcuts in the corresponding program.
- You can define shortcuts in kitty to send another key combination.

If you are using ssh and the keys are not working properly, upgrade to the latest release version and make sure `TERMINFO` is installed correctly.

https://sw.kovidgoyal.net/kitty/faq/

**Reply by ForteDexe** [reply]
Thank you, you are right, the problem is the shell :).

**Comment by cerodah** [comment]
Has it been solved?

Thanks.

**Comment by lalishansh** [comment]
@ForteDexe, how did you do it ??

**Comment by LUISVILL02** [comment]
Solved? How did you configured the shell?
